src/main/java/com/company/Tweet.java

```diff
@@ -17,15 +20,13 @@ public static void main(String[] args) {
         TwitterFactory tf = new TwitterFactory(cb.build());
         Twitter twitter = tf.getInstance();
         try{
-            if(args[0].length()>280){
-                throw new IndexOutOfBoundsException();
+            if(args == null || args.length()==0 || args.trim().isEmpty() || args.length()>280){
```

Comment:
you can simplify this using and && statement and reduce the number of checks using StringUtils.isEmpty